diff --git a/nemo_rl/models/automodel/setup.py b/nemo_rl/models/automodel/setup.py
index c1799ca..75db1bc 100644
--- a/nemo_rl/models/automodel/setup.py
+++ b/nemo_rl/models/automodel/setup.py
@@ -327,7 +327,7 @@ def setup_policy_model(
     # Load model config
     model_config = AutoConfig.from_pretrained(
         model_name,
-        torch_dtype=torch.float32,  # Always load in float32 for master weights
+        dtype=torch.float32,  # Always load in float32 for master weights
         trust_remote_code=True,
         attn_implementation="flash_attention_2" if enable_seq_packing else None,
         **hf_config_overrides,
